### PR TITLE
docs(linux): update headless ssh guide

### PR DESCRIPTION
**Pull request by e-dong**
## Description

* Replace uinput sudoers hack with better udev rules (user needs to be in input group)
* Added some info how to configure the DPI to better scale the streamed virtual display on the client device.

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

File: docs/source/about/guides/app_examples.rst [hunk 1]
```diff
@@ -206,6 +206,9 @@ Changing Resolution and Refresh Rate
       |                      | Undo: ``xrandr --output HDMI-1 --mode 3840x2160 --rate 120``                                                                       |
       +----------------------+------------------------------------------------------------------------------------------------------------------------------------+
 
+      .. warning::
```
Comment (ReenigneArcher) [inline docs/source/about/guides/app_examples.rst:209]:
```suggestion
      .. attention::
```

A warning may be a bit extreme.

File: docs/source/about/guides/app_examples.rst [hunk 2]
```diff
@@ -247,6 +250,13 @@ Changing Resolution and Refresh Rate
 
                # Optional reset your wallpaper to fit to new resolution
                # xwallpaper --zoom /path/to/wallpaper.png
+         
+         .. note::
+            Using ``xrandr --scale`` may lead to poor results like blurriness. It is preferred to adjust the DPI instead.
+
+            reference:
+              - https://wiki.archlinux.org/title/xorg
+              - https://winaero.com/find-change-screen-dpi-linux/
```
Comment (ReenigneArcher) [inline docs/source/about/guides/app_examples.rst:259]:
```suggestion
            **reference:**

               - https://wiki.archlinux.org/title/xorg
               - https://winaero.com/find-change-screen-dpi-linux/
```

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 3]
```diff
@@ -12,7 +12,8 @@ This is a guide to setup remote SSH into host to startup X server and sunshine w
 The virtual display is accelerated by the NVidia GPU using the TwinView configuration.
 
 .. attention::
-    This guide is specific for Xorg and NVidia GPUs. I start the X server using the ``startx`` command.
+    This guide is specific for Xorg and NVidia GPUs. This guide assumes the host is not using a desktop environment and only a barebones windows manager.
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:15]: What is a "barebones" window manager? Can you provide an example?

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 4]
```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
...
 
-      echo "rfkill block $(rfkill list | grep "Wireless LAN" \
-        | sed 's/^\([[:digit:]]\).*/\1/')" >> sunshine-setup.sh
-
-**Sudo Configuration**
-
-We will manually change the permissions of ``/dev/uinput`` using ``chown``.
-You need to use ``sudo`` to make this change, so add/update the entry in ``/etc/sudoers.d/${USER}``
+      echo 'KERNEL=="uinput", SUBSYSTEM=="misc", MODE="0660", GROUP="input", OPTIONS+="static_node=uinput"' | \
+      sudo tee /etc/udev/rules.d/85-sunshine.rules
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:266]: Should we just update our rules so they work in this situation as well?

Here -> https://github.com/LizardByte/Sunshine/blob/nightly/src_assets/linux/misc/85-sunshine.rules

And anywhere it occurs in the docs?

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 5]
```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
...
-
-.. warning::
-   Be very careful of messing this config up. If you make a typo, *YOU LOSE THE ABILITY TO USE SUDO*.
-   Fortunately, your system is not borked, you will need to login as root to fix the config.
-   You may want to setup a backup user / SSH into the host as root to fix the config if this happens.
-   Otherwise you will need to plug your machine back into a monitor and login as root to fix this.
-   To enable root login over SSH edit your SSHD config, and add ``PermitRootLogin yes``, and restart the SSH server.
-
-#. First make a backup of your ``/etc/sudoers.d/${USER}`` file.
+#. Add the user running sunshine to the `input` group.
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:268]: This used to be part of our instructions... not sure when or why it got removed.

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 6]
```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
...
 
    .. code-block:: bash
 
-      echo "${USER} ALL=(ALL:ALL) ALL, NOPASSWD: $(pwd)/sunshine-setup.sh" \
-        | sudo tee /etc/sudoers.d/${USER}
-
-These changes allow the script to use sudo without being prompted with a password.
-
-e.g. ``sudo $(pwd)/sunshine-setup.sh``
+      sudo udevadm control --reload-rules && sudo udevadm trigger
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:279]: Nice! I didn't know about these commands... always just rebooted.

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 7]
```diff
@@ -250,125 +249,102 @@ As an alternative to a dummy dongle, you can use this config to create a virtual
...
-    # e.g.
-    # steam &> /dev/null &
-    # firefox &> /dev/null &
-    # kdeconnect-app &> /dev/null &
+This is what my .xinitrc file looks like:
+
+.. code-block:: bash
+
+   xrdb -merge ~/.Xresources
+   ...
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:343]: Are the `...` literal? If not someone will take them literally.

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 8]
```diff
@@ -476,42 +452,66 @@ This can be run on Unix systems, or on Windows using the ``git-bash`` or any bas
 For Android/iOS you can install Linux emulators, e.g. ``Userland`` for Android and ``ISH`` for iOS.
 The neat part is that you can execute one script to launch Sunshine from your phone or tablet!
 
+Let's call this script ``start-stream.sh``.
+
 .. code-block:: bash
 
    #!/bin/bash
 
-   ssh_args="<user>@192.168.X.X" # Or use alias set in ~/.ssh/config
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:483]: Why was the comment removed?

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 9]
```diff
@@ -476,42 +452,66 @@ This can be run on Unix systems, or on Windows using the ``git-bash`` or any bas
...
+``ipad.sh``
+
+.. code-block:: bash
+
+   #!/bin/bash
+   /path/to/start-stream.sh 180
+
+DPI References:
+  - https://wiki.archlinux.org/title/xorg
+  - https://winaero.com/find-change-screen-dpi-linux/
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:513]:
```suggestion
**DPI reference:**

  - https://wiki.archlinux.org/title/xorg
  - https://winaero.com/find-change-screen-dpi-linux/
```

File: docs/source/about/guides/linux/headless_ssh.rst [hunk 10]
```diff
@@ -522,5 +522,5 @@ If you have any feedback and any suggestions, feel free to make a post on Discor
 
 .. seealso::
    Now that you have a virtual display, you may want to automate changing the resolution
-   and refresh rate prior to connecting to an app. See :ref:`Changing Resolution and Refresh Rate
+   and refresh rate on the host prior to connecting to an app. See :ref:`Changing Resolution and Refresh Rate
```
Comment (ReenigneArcher) [inline docs/source/about/guides/linux/headless_ssh.rst:525]:
```suggestion
   and refresh rate, on the host, prior to connecting to an app. See :ref:`Changing Resolution and Refresh Rate
```

**Review by ReenigneArcher — CHANGES_REQUESTED** [review]
I'm wondering if some of this should be changed in our code (udev rules), and others added back to our instructions (add user to input group)...

File: docs/source/about/guides/linux/headless_ssh.rst (hunk 7 above)
Comment (e-dong) [reply docs/source/about/guides/linux/headless_ssh.rst:343]: lol

**Pull request closed without merging**